src/james.format.xml.clss.pas

```diff
@@ -39,20 +39,24 @@ interface
 
 type
   {$IFDEF FPC}
-  TXMLDocument = Laz2_DOM.TXMLDocument;
-  TDOMNode = Laz2_DOM.TDOMNode;
+    TXMLDocument = Laz2_DOM.TXMLDocument;
+    TDOMNode = Laz2_DOM.TDOMNode;
   {$ENDIF}
 
   TXMLComponent = class
```

Comment:
This class is not good by the way. Its name is wrong too.
We need to have our own `IXMLDocument` and `TXMLDocument` which can be `IXocument` and `TXocument`. These artefacts will be the transition between FPC/Lazarus and Delphi.

What do you think?

PS. But for now, this is Ok.